src/stores/profile.store.ts

```diff
@@ -18,15 +18,12 @@ export const useProfileStore = defineStore('profile', {
         return;
       }
     },
-    async findProfiles(keyword: string) {
-      return ((await firebaseService.findAll('profiles')) as IProfile[]).filter(
-        (profile) => {
-          return (
-            profile.type == 'adviser' &&
-            profile.fullName.toLowerCase().match(keyword.toLowerCase())
-          );
-        }
-      );
```

Comment:
avoid removing others code of ther developers

Reply:
My apologies, sir. That was my mistake. I suggested to the Good Bye team, specifically Junaid, that he could modify it to a stream. I was the one who advised Ate Intishar (@elsa17nam) to modify or, in this case, exclude it. It won't happen again.

Reply:
I reverted it as it is being used in SearchBarComponent